examples/plotting_with_geoplot.py

```diff
@@ -15,74 +15,84 @@
 First we'll load in the data using GeoPandas.
 """
 import geopandas
+import geoplot
 
-path = geopandas.datasets.get_path('naturalearth_lowres')
-df = geopandas.read_file(path)
-# Add a column we'll use later
-df['gdp_pp'] = df['gdp_md_est'] / df['pop_est']
-
-boroughs = geopandas.read_file(geopandas.datasets.get_path('nybb')).to_crs(epsg='4326')
-injurious_collisions = geopandas.read_file(
-    "https://github.com/ResidentMario/geoplot-data/raw/master/nyc-injurious-collisions.geojson")
-
+world = geopandas.read_file(
+    geopandas.datasets.get_path('naturalearth_lowres')
+)
+boroughs = geopandas.read_file(
+    geopandas.datasets.get_path('nyc_boroughs')
+)
+collisions = geopandas.read_file(
+    geopandas.datasets.get_path('nyc_injurious_collisions')
```

Comment:
Yes, it should. 😅

I didn't rerun the script after my last change. Shame on me for failing to notice this bad copy-paste job. If you want to address this yourself that's good, otherwise I can do a quick fixer-upper PR myself.

Reply:
And shame on me that I didn't notice either and merged.

Reply:
No, no, don't blame yourself too much, just a small error :-)

Reply:
Already done https://github.com/geopandas/geopandas/pull/1046